packages/tupaia-web/src/features/Visuals/View/MultiValueRow.tsx

```diff
@@ -4,19 +4,19 @@
  */
 import React from 'react';
 import styled from 'styled-components';
-import { MultiValueRowViewConfig, ViewConfig } from '@tupaia/types';
+import { MultiValueRowViewConfig } from '@tupaia/types';
 import {
   Table,
   TableCell as MuiTableCell,
   TableRow,
   TableBody,
   TableHead,
 } from '@material-ui/core';
-import { ViewDataItem } from '../../../types';
+import { ViewDataItem, ViewReport, DashboardItemType } from '../../../types';
 
 interface MultiValueRowProps {
-  data?: ViewDataItem[];
-  config: ViewConfig;
+  report: ViewReport;
+  config: DashboardItemType;
```

Comment:
```suggestion
  config: DashboardItemType['config'];
```
Maybe this could justify another type for the config for convenience?